src/Repository/CitizenProjectRepository.php

```diff
@@ -120,6 +120,43 @@ public function searchAll(SearchParametersFilter $search): iterable
         ;
     }
 
+    /**
+     * Finds all citizen projects around an Adherent.
+     *
+     * @param Adherent $adherent
+     * @param int      $limit    Limit of citizen project
+     * @param int      $maxAge   Duration in days
+     *
+     * @return array List of CitizenProject
```

Comment:
`@return array List of CitizenProject` => `@return CitizenProject[]`

but these docblocks are not very usefull imho

Reply:
Specialized arrays return do help the IDE and avoid adding "duck type hint" in the code when manipulating entries (i.e in foreach loops).